package voyage;

import "texture_atlas.proto";

message TileMapCfg {
	repeated TileMapLayerCfg layers=1;
	required uint32 size_x=2;
	required uint32 size_y=3;	
}

message TileMapLayerCfg {
	required string data_file=1;
	required uint32 shader_id=2;
	required uint32 atlas_id=3;	
}